packages/grafana-ui/src/components/Menu/Menu.story.tsx

```diff
@@ -24,20 +24,18 @@ const meta: Meta<typeof Menu> = {
     actions: {
       disabled: true,
     },
-    // TODO fix a11y issue in story and remove this
-    a11y: { test: 'off' },
   },
 };
 
 export function Examples() {
   return (
-    <Stack direction="column">
+    <Stack direction="column" width="fit-content">
       <StoryExample name="Plain">
         <Menu>
           <Menu.Item label="Google" />
           <Menu.Item label="Filter" />
           <Menu.Item label="Active" active />
-          <Menu.Item label="I am a link" url="http://google.com" target="_blank" />
+          <Menu.Item label="I am a link" url="http://google.com" target="_blank" role="menuitem" />
```

Comment:
I might have misunderstood the discussion but they were not saying `If there is a URL, will use the passed role, or fallback to undefined, which preserves role = link`